### PR TITLE
Credit creators, using same name across modules to avoid split statistics

### PR DESCRIPTION
**Pull request by yajo**
I'm trying to unify Grupo ESOC work into one single name. Could you merge this please?

### CODE REVIEW

File: partner_firstname/__openerp__.py [hunk 1]
```diff
@@ -22,7 +22,9 @@
     'name': 'Partner first name and last name',
     'summary': "Split first name and last name for non company partners",
     'version': '8.0.2.1.0',
-    'author': "Camptocamp, Grupo ESOC, Odoo Community Association (OCA)",
+    "author": "Camptocamp, "
+              "Grupo ESOC Ingeniería de Servicios, "
```
Comment (pedrobaeza) [inline partner_firstname/__openerp__.py:26]: Shouldn't you include here Antiun?

File: partner_second_lastname/__openerp__.py [hunk 2]
```diff
@@ -4,12 +4,14 @@
 
 {
     "name": "Partner second last name",
+    "summary": "Have split first and second lastnames",
     "version": "8.0.4.0.0",
-    "author": "Grupo ESOC, Odoo Community Association (OCA)",
     "license": "AGPL-3",
+    "website": "https://grupoesoc.es",
+    "author": "Grupo ESOC Ingeniería de Servicios, "
```
Comment (pedrobaeza) [inline partner_second_lastname/__openerp__.py:11]: And also here?

File: partner_firstname/__openerp__.py (hunk 1 above)
Comment (yajo) [inline partner_firstname/__openerp__.py:26]: Their contribution is in https://github.com/OCA/partner-contact/pull/227, not merged yet.

File: partner_second_lastname/__openerp__.py (hunk 2 above)
Comment (yajo) [inline partner_second_lastname/__openerp__.py:11]: Same here AFAIK.

**Comment by antespi** [comment]
:+1: good for me, I'll rebase this in #227

File: partner_firstname/__openerp__.py (hunk 1 above)
Comment (antespi) [inline partner_firstname/__openerp__.py:26]: I'll rebase it in #227

**Comment by pedrobaeza** [comment]
OK then

**Pull request merged**